cmd/osv-scanner/main.go

```diff
@@ -159,6 +167,14 @@ func run(args []string, stdout, stderr io.Writer) int {
 				return err
 			}
 
+			var callAnalysisStates map[string]bool
+			if context.Bool("experimental-call-analysis") {
```

Comment:
You should check if the flag is set since you can explicitly provide booleans i.e. this won't print the warning for `--experimental-call-analysis=false`

Reply:
Thanks!